01/app.js

```diff
@@ -1,2 +1,15 @@
 const randomNumber = Math.round(Math.random() * 20);
-console.log(randomNumber);
+console.log('Wylosowana liczba to ' + randomNumber);
+
+if (randomNumber > 5) {
+    for (let i = 5; i <= randomNumber; i++) {
+        if (i <= randomNumber) {
```

Comment:
Ten warunek jest niepotrzebny - masz go już w pętli `for`

Reply:
wydaje mi się że jednak jest potrzebny. Są 2 opcje: (1) wylosowana liczba jest większa od 5 (wówczas rozpoczynamy pętlę od 5 i kończymy na wylosowanej liczbie (od 6 do 20), (2) wylosowana liczba jest mniejsza równa lub mniejsza od 5 (wówczas wyświetla komunikat)